libs/core/src/lib/wizard/wizard.component.html

```diff
@@ -2,7 +2,7 @@
     <ng-content select="fd-wizard-navigation"></ng-content>
     <div
         *ngIf="appendToWizard"
-        class="fd-wizard-container-wrapper"
+        class="fd-wizard-container-wrapper fd-scrollbar"
```

Comment:
Instead of direct applying of CSS class, I would recommend to use additional element with `fd-scrollbar` directive applied to it.

The first reason is that to see the styles of that CSS class, we should have those imported somewhere globally, just like icons.

The second reason is more my opinion that in NGX we work on the level of directives and components and not on classes.